src/bikeracks/static/main.js

```diff
@@ -46,14 +46,20 @@ function subForm (e){
     });
 }
 
-function deleteForm(e) {
-    e.preventDefault();
-    
+const $suggestRemoval = $('#suggestRemoval');
+const $submitFeedback = $('#submitFeedback');
 
+function submitRemovalForm(e) {
+    e.preventDefault();
+    const button = $(e.target),
+    rack_id = button.data('rack_id');
+    console.log(e);
+    console.log(rack_id);
```

Comment:
Remove these random console.log statements?

Reply:
I will be putting them right back in the next pull request though.